docs/getting-started/dev-portal/react.md

```diff
@@ -4,17 +4,21 @@ Learn how to quickly get started with Elements Dev Portal in a React project.
 
 ## Create React App Template
 
-We've created a Create React App template, which allows you to create a brand new Elements Dev Portal website in React without any additional setup.
+We've created a [Create React App template](https://github.com/stoplightio/cra-template-elements-dev-portal), which allows you to create a brand new Elements Dev Portal website in React without any additional setup.
 
-Simply run:
```

Comment:
We should try and avoid saying "simple" "easy" "just" because it can make people feel bad when they don't find it simple. This is a common mistake lots of us make but we try and avoid it on docs and customer facing stuff as much as possible.